packages/e2e-tests/specs/performance/performance.test.js

```diff
@@ -122,8 +122,16 @@ describe( 'Performance', () => {
 			);
 		}
 
+		// Wait for all the blocks to be async rendered.
+		// eslint-disable-next-line no-restricted-syntax
+		await page.waitFor( 20000 );
```

Comment:
This seems kind of random. Can we use `waitForFunction`?

Reply:
yes, we can probably find a better replacement, I just didn't figure it out yet :)